packages/config/src/lib/createConfig.ts

```diff
@@ -117,7 +119,7 @@ export const createConfig = ({
   const devServerPort = typeof port === 'number' ? port : useProxy || standalone ? 1337 : 8002;
   return {
     mode: mode || (isProd ? 'production' : 'development'),
-    devtool: false,
+    devtool: devtool || false,
```

Comment:
The default value is set in the argument now you can simply use `devtool: devtool,` or just `devtool,`.

Reply:
I got a bit too caught up in the meeting, thank you!